com.woltlab.wcf/templates/pageFooterCopyright.tpl

```diff
@@ -1,3 +1,3 @@
 {event name='copyright'}
 
-{if !'WOLTLAB_BRANDING'|defined || WOLTLAB_BRANDING}<div class="copyright">{lang}wcf.page.copyright{/lang}</div>{/if}
+{if (!'WOLTLAB_BRANDING'|defined || WOLTLAB_BRANDING) && $showWoltLabBranding}<div class="copyright">{lang}wcf.page.copyright{/lang}</div>{/if}
```

Comment:
Check whether `showWoltLabBranding` is defined (and assume `$showWoltLabBranding = true` if it is not). I'm fairly sure that this will crash when a `NamedUserException` is thrown.